examples/alphabet/alphabet.ino

```diff
@@ -0,0 +1,17 @@
+#include "segm8.h"
```

Comment:
Please add a brief description of the experiment at the top.

Reply:
done